.github/workflows/ci_cd.yml

```
@@ -49,3 +49,24 @@ jobs:
     - name: Run implemented Acceptance tests
       working-directory: ./http_server_spec
       run: rake test:f0
+
+  deploy:
+
+    if: github.ref == 'refs/heads/main'
+    needs: build
+    name: Deploy to Production 
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: appleboy/ssh-action@master
+      with:
+        host: ${{ secrets.HOST }}
+        username: ${{ secrets.USERNAME }}
+        key: ${{ secrets.SSH_KEY }}
+        script: |
+          cd build
+          git -C HTTP-Server pull || git clone git@github.com:ksdenisova/HTTP-Server.git
+          cd HTTP-Server
+          kill -9 $(lsof -t -i:5000)
+          mix deps.get
+          elixir --erl "-detached" -S mix run --no-halt
```

This may be better served as its own script file.
```suggestion
        script: scripts/deploy.sh
```

Also, think if there would be other useful scripts you could use (like starting and stoping the server).  Just a suggestion.

---

This is a good idea. Scripts should be stored on the production server?

---

they can be, or alternatively in a `scripts/` folder on the repo. Your choice.